Server/mods/deathmatch/logic/CResourceFile.h

```diff
@@ -37,12 +37,22 @@ class CResourceFile
         RESOURCE_FILE_TYPE_NONE,
     };            // TODO: sort all client-side enums and use >= (instead of each individual type) on comparisons that use this enum?
 
+    enum class eResourceCategory
+    {
+        ALL = 0,
+        SCRIPTS = 1,
+        MAPS = 2,
+        CONFIGS = 3,
+        FILES = 4,
+    };
+
```

Comment:
What is the purpose of setting values ​​here when on the client side the exact same enum has no values ​​set?

Reply:
Forgot to remove it